src/app/src/main/kotlin/org/imperial/mrc/hint/clients/FuelClient.kt

```diff
@@ -87,4 +93,30 @@ abstract class FuelClient(protected val baseUrl: String)
                 .second
                 .asResponseEntity()
     }
+
+    private fun getRequest(uri: URI): HttpRequest
+    {
+        val entry = standardHeaders().entries.first()
```

Comment:
Do you not want to include all headers here?

Reply:
Yeah, what's going on here?